eslint-plugin/.eslintrc.js

```diff
@@ -1,6 +1,6 @@
 /*
- * ecoCode JavaScript plugin - Provides rules to reduce the environmental footprint of your JavaScript programs
- * Copyright © 2023 Green Code Initiative (https://www.ecocode.io)
+ * creedengo JavaScript plugin - Provides rules to reduce the environmental footprint of your JavaScript programs
+ * Copyright © 2023 Green Code Initiative (https://green-code-initiative.org)
```

Comment:
why do you keep 2023 ? maybe upgrade 2024 in pom.xml and launch licence maven plugin to improve this date

Reply:
I do not choose to keep it, because it is based on the `inceptionYear` property inside the `pom.xml` file.
Based on [the Maven documentation here](https://maven.apache.org/pom.html#:~:text=project%27s%20home%20page.-,inceptionYear,-%3A%20The%20year%20the), this value represents the year when the project started, so I do not think I should update it 🤷‍♂️ 

Do you consider I should change this value because of the move to Creedengo?

Reply:
No you're ... we should follow maven best practice ... but I'm wrong with other projects because I changed it :( ... I will revert it soon, so :p